extensions/wikia/ARecoveryEngine/js/PageFair/pageFair.js

```diff
@@ -33,10 +33,10 @@ define('ext.wikia.aRecoveryEngine.recovery.pageFair', [
 		return isBlocking;
 	}
 
-	function addMarker(slotId) {
-		var slot = document.getElementById(slotId);
-
-		slot.setAttribute('adonis-marker', '');
+	function addMarker(slotElement) {
+		if (slotElement) {
```

Comment:
Let's remove it then and ask @bulaszenko to do his best to find a way for throwing the error ;) A weak compromise is to add `else` with logging inside. So, once we notice no markers added when all conditions are met, we'll add logging params and see if a lack of element is an issue.